decentraland/unity-explorer
# Fix: fixed a series of broken meta files

## What does this PR change?

In order to fix #2390 loads of meta files have been regenerated to avoid an issue caused by broken meta files that contained changes caused by merge conflicts

...

## How to test the changes?



1. Launch the explorer
2. Do a quick smoke test to verify that it loads fine
3. Check backpack and map as well
4. Check chat

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
